Kitodo/src/main/java/org/kitodo/production/forms/massimport/MassImportForm.java

```diff
@@ -54,9 +56,10 @@ public class MassImportForm extends BaseForm {
     private ImportConfiguration importConfiguration;
     private UploadedFile file;
     private String csvSeparator = ";";
-    private String previousCsvSeparator = null;
     private List<String> metadataKeys = new LinkedList<>(Collections.singletonList("ID"));
     private List<CsvRecord> records = new LinkedList<>();
+    private String importedCsvHeaderLine = "";
+    private List<String> importedCsvLines = new LinkedList<>();
```

Comment:
Should `importedCsvHeaderLine` and `importedCsvLines` also be reset in `resetValues`, perhaps? Otherwise, these values could remain after consecutive file uploads of empy CSV files, couldn't they? Or is that perhaps even intended?

Reply:
I agree. It is more consistent when to reset those values as well.